static/docs/commands-reference/diff.md

```diff
@@ -13,9 +13,9 @@ usage: dvc diff [-h] [-q | -v] [-t TARGET] a_ref [b_ref]
 
 positional arguments:
   a_ref                 Git reference from which diff calculates
-  b_ref                 Git reference until which diff calculates,
-                        if omitted diff shows the difference
-                        between current HEAD and a_ref
+  b_ref                 Git reference untill which diff calculates, if
```

Comment:
typo in untill?

Reply:
Good catch. It's copy paste from the actual command usage though.

I've fixed it along with other usage text problems in my last commit (65c12c4) and opened https://github.com/iterative/dvc/issues/2169 to also update the core repo.